src/.pre-commit-config.yaml.jinja

```
@@ -176,11 +178,14 @@ repos:
       - id: check-xml
       - id: mixed-line-ending
         args: ["--fix=lf"]
+  {%- if not use_ruff or odoo_version < 15.0 %}  # ruff doesn't support python 3.6
```

AFAICS in the copier questionary, you cannot use ruff in Odoo versions lower than 15, so this or is redundant:

```suggestion
  {%- if not use_ruff %}  # ruff doesn't support python 3.6
```

---

The copier question accepts ruff for 14.

---

Ah true.